docs/01_setup.md

```diff
@@ -133,6 +105,67 @@ pnpm deploy-poll:NETWORK
 
 See [MACI docs](https://maci.pse.dev/docs/v1.2/integrating#deployment) for more information.
 
+
+## 3. Configuration
+
+The `.env.example` file contains instructions for most of these steps.
+
+At the very minimum you need to configure a subgraph, admin address, MACI address, the EAS Schema and the voting periods under App Configuration.
+
+#### Subgraph
+
+In the MACI repo and head to the subgraph folder.
+
+> [!IMPORTANT]
+> If you are using the v1.2.4 branch, you will need to checkout to dev and build in order to deploy a subgraph, once deployed remember to switch back to v1.2.4 in order to finalize the round. The following steps focus on The Graph, but you can deploy a subgraph using Alchemy.
+
+1. Make sure you have `{network}.json` file in `config` folder, where network is a CLI name supported for subgraph network [https://thegraph.com/docs/en/developing/supported-networks/](https://thegraph.com/docs/en/developing/supported-networks/).
+
+2. Create subgraph in [the graph studio](https://thegraph.com/studio/).
+
+3. Add network, maci contract address and maci contract deployed block.
+
+```json
+{
+  "network": "optimism-sepolia",
+  "maciContractAddress": "0xD18Ca45b6cC1f409380731C40551BD66932046c3",
+  "maciContractStartBlock": 11052407
+}
+```
+
+4. Run `pnpm run build`. You can use env variables `NETWORK` and `VERSION` to switch config files.
+    - Create an `.env` file, and run the follow command in the console `export $(xargs < .env)`
+5. Run `graph auth --studio {key}`. You can find the key in subgraph studio dashboard.
+6. Run `pnpm run deploy` to deploy subgraph
+
+#### Network
+
+The default configuration is Optimism Sepolia for development and Optimism for production.
+
+You can find supported networks on the EAS documentation website: https://docs.attest.sh/docs/quick--start/contracts
+
+#### App
+
+Configure how many votes each voter receives and how many votes each project can receive.
```

Comment:
not sure about this part "how many votes each project can receive" - that's up to the voter, they could spend max credits on one single project. Perhaps you meant something else?

Reply:
inside `.env.example` there are parameters as below:
```
# How many votes each ballot can hold
NEXT_PUBLIC_MAX_VOTES_TOTAL=150
# How many votes each project can receive
NEXT_PUBLIC_MAX_VOTES_PROJECT=50
```

thinking about removing this part also in this PR? or we could open another PR for removing this?

Reply:
I'll remove here